src/controllers/hid/hidcontroller.cpp

```diff
@@ -219,6 +219,7 @@ int HidController::open() {
     for (int i = 0; i < kNumBuffers; i++) {
```

Comment:
Isn't this initialization obsolete now?

Reply:
it's a one-time memset, it seems worth initializing values to zero, esp because we pass these buffers to javascript. Better safe than sorry here.

Reply:
I am not arguing about runtime overhead. This is completely redundant code and we never pass any uninitialized buffer to JavaScript. A comment in the code should clarify that. Otherwise, a future developer might wonder why this code is needed.

Reply:
if the read is <255 bytes, wouldn't the first version sent to javascript be bytesRead of data, and 255-bytesRead of junk?  I'll add a comment, but leave it in

Reply:
The size of the array is always limited by the number of bytes read.